src/lib/snark_params/snark_params.ml

```diff
@@ -472,8 +472,20 @@ module Tick = struct
         let fold = gen_fold n in
         let result, unchunked_result = run_updates fold in
         assert (equal_states result unchunked_result)
+
+      let hash_unchunked n =
+        ignore (Lazy.force chunk_table) ;
+        let fold = gen_fold n in
+        State.update_fold_unchunked initial_state fold
+
+      let hash_chunked n =
+        ignore (Lazy.force chunk_table) ;
```

Comment:
It'd be better to use `bench_fun` as describe [here](https://github.com/janestreet/ppx_bench/blob/f267f868b6e666899ab4abf771e6db8842c8cb5a/example/ppx_bench_sample.ml#L29) to set up the chunk table.

Reply:
OK, done.